devicetypes/smartthings/smartweather-station-tile.src/smartweather-station-tile.groovy

```diff
@@ -22,6 +22,9 @@ metadata {
         capability "Temperature Measurement"
         capability "Relative Humidity Measurement"
         capability "Ultraviolet Index"
+		capability "Wind Speed"
+        capability "stclassic.Wind"
```

Comment:
Why did you choose `stclassic` as the namespace? Seems a little odd since the point is to get these working in the new app.

Reply:
I picked it because I really couldn't think of something better (other than maybe stlegacy) that also wouldn't risk conflicting with another namespace. The thinking was "these are being used by device handlers from the classic/legacy platform." Though I am open to suggestions/ideas.

Reply:
I included the custom capability as I have defined it. I'll note elsewhere the capabilities that I plan to create and add in another PR.

Reply:
How about something more specific to this DTH like `st-weatherstation`?

@greens @varzac @BroderickCarlin might have opinions too

Reply:
I don't really like `stclassic`, but I don't have a better idea.

Reply:
Since custom namespaces are closing, I figured I'd snag something more general that can be added to over time.

Reply:
@tpmanley I liked the idea about a DTH-specific namespace. Originally I was against it, but it at least compartmentalized the capabilities. I opted to break out in to multiple capabilities instead of just a DTH-specific capability. I also created the namespace `stcustom` for future use.